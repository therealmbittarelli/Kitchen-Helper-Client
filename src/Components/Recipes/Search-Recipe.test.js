import React from 'react';
import ReactDOM from 'react-dom';
import SearchRecipe from './Search-Recipe';
import { BrowserRouter } from 'react-router-dom';
import renderer from "react-test-renderer";

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<BrowserRouter>
    <SearchRecipe />
  </BrowserRouter>, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders the UI as expected', () => {
  const tree = renderer.create(<BrowserRouter>
    <SearchRecipe />
  </BrowserRouter>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});