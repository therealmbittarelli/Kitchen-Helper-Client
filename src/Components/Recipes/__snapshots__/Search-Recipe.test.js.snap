// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="searchRecipe"
>
  <p
    id="larger-search-text"
  >
    Search our collection of recipes!
  </p>
  <p
    id="recipe-search-text"
  >
    For best results, please use whole words as search terms. 
    <br />
    Feel free to include more than one search term at a time.
  </p>
  <br />
  <form
    className="searchArea"
    onSubmit={[Function]}
  >
    <input
      id="searchBar"
      onChange={[Function]}
      placeholder="Search by ingredients"
      type="text"
    />
    <button
      id="searchButton"
      type="submit"
    >
      Search
    </button>
  </form>
  <section
    className="recipeResults"
  />
</div>
`;
